apply-pr.cmd

```diff
@@ -0,0 +1,5 @@
+REM %1 - PR number
+
+git checkout master
+git fetch upstream pull/%1/head:PR%1
+git checkout PR%1

```

Comment:
Hi Tim, I guess this was added unintentionally?

Reply:
There's some other .cmd files added as well, that I guess was also added unintentionally

Reply:
Thanks, @dybber; those must have been caught up in a housekeeping sweep. I've unversioned/ignored them now